client.go

```diff
@@ -334,10 +304,7 @@ func (c *Client) keepAliveSender() {
 		}
 
 		c.conn.SetWriteDeadline(time.Now().Add(5 * time.Second))
-		if err := c.conn.WriteJSON(frame{
-			ModuleID: moduleIDUSP,
-			Messages: []interface{}{},
-		}); err != nil {
+		if err := c.conn.WriteJSON(map[string]interface{}{}); err != nil {
```

Comment:
The keepalive is completely empty?

Reply:
Yup, just to test the connection and prevent positive confirmation we're still there.